k8s/aat/common/aac/manage-case-assignment.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: aac-manage-case-assignment
-    version: 0.0.5
+    version: 0.0.6
```

Comment:
it's recommended to swap to github source:
https://github.com/hmcts/hmcts-charts/tree/master/stable/aac-manage-case-assignment

https://hmcts-reform.slack.com/files/T1L0WSW9F/F018A7PQBNC?origin_team=T1L0WSW9F

Reply:
Thanks Tim.